docs-src/python/choose-dev-cluster.md

```diff
@@ -0,0 +1,133 @@
+---
+id: choose-dev-cluster
+title: Choose a development Cluster
+sidebar_label: Choose dev Cluster
+description: Discover which development Cluster you should choose
+tags:
+  - developer guide
+  - python sdk
+---
+
+**Which development Cluster should you choose?**
+
+We recommend choosing a development environment based on your requirements.
+
+The source code for the Temporal Server (the orchestrating component of the Temporal Cluster) is licensed under the MIT open source license. So, in theory, anyone can take the Temporal Server code and run their Temporal Platform in any number of creative ways.
+
+However, for most developers we recommend starting by choosing one of the following:
+
+- [Local development server](#local-dev-server)
+- [Temporal Cloud](#temporal-cloud)
+- [Self-hosted Temporal Cluster](#self-hosted-temporal-cluster)
+
+:::info Temporal does not directly run your code
+
+Keep in mind that in every scenario, the “Temporal Platform” does not host and run your Workers (application code).
```

Comment:
In some docs you use `“` style quotes, and in some `"`

Reply:
Removing smart quotes.